env.example

```diff
@@ -20,3 +20,5 @@ INTERNAL_API_BASE_URL=http://localhost:9696
 INTERNAL_SYNC_API_BASE_URL=http://localhost:9697
 NEXT_PUBLIC_APP_ENVIRONMENT="development"
 DEFAULT_SYNC_DAYS=31
```

Comment:
I think this is also left to be added to Dockerfiles from the last effort

Reply:
Can be done in another PR though

Reply:
I added it as args to the Dockerfiles